instrumentation/servlet/servlet-2.3/build.gradle.kts

```diff
@@ -9,12 +9,12 @@ muzzle {
     pass {
         group = "javax.servlet"
         module = "servlet-api"
-        versions = "[2.3,)"
+        versions = "[2.3, 3.0)"
     }
     fail {
         group = "javax.servlet"
         module = "javax.servlet-api"
-        versions = "(,)"
+        versions = "(3.0.1,)"
```

Comment:
This shows a regression in the muzzle. Without instrumentation modules it was passing with the previous settings.